testing/ncids-css-testing/stories/components/nci-header/nci-header-primary-gap.jsx

```diff
@@ -0,0 +1,428 @@
+// MegaMenu HTML Section language=HTML
+const MegaMenu = `
```

Comment:
either move the example with longer header to `nci-header-primary.js` or move both mega menu examples to their own file